mockup_package/mockup/image_generator.py

```diff
@@ -190,15 +190,16 @@ def create_mockup_image(self, device_path, device_mask_path_prefix, result_path)
             tmp = self.target_points[0].copy()
             self.target_points[0] = self.target_points[1]
             self.target_points[1] = tmp
+            shifted_points = self.target_points + np.array([PADDING, PADDING])
```

Comment:
question: How does adding padding and shifting points not affect the pictures fitting the phone?

e.g. if imageA fits perfectly on a phone (say moto-e) before. after adding padding, does imageA still fits perfectly? If so, how?

Thanks~

Reply:
This shows why it become incorrect before this update:
![image](https://github.com/user-attachments/assets/c751f8a2-4ff5-4a53-8eff-d58432b2f48a)

in #79 we adjusted the position of the pasted image if `is_mockup_image_at_front`, but forgot to update the other cases

Reply:
Thanks yayun! This explanation looks amazing. Can you put it in `mockuphone.com/docs`? 🙇

Reply:
@pkong-ds add docs to explain why we add padding when generating mockup ~